persistence/src/test/scala/za/co/absa/spline/persistence/tx/JSTxBuilderSpec.scala

```diff
@@ -19,40 +19,41 @@ package za.co.absa.spline.persistence.tx
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 import org.scalatestplus.mockito.MockitoSugar
-import za.co.absa.spline.persistence.model.NodeDef
+import za.co.absa.spline.persistence.model.{ArangoDocument, NodeDef}
 
-class TxBuilderSpec extends AnyFlatSpec with Matchers with MockitoSugar {
+class JSTxBuilderSpec extends AnyFlatSpec with Matchers with MockitoSugar {
 
   behavior of "generateJs()"
 
   it should "generate NATIVE statements" in {
-    val generatedJS = new TxBuilder()
-      .addQuery(NativeQuery("db.FOO();"))
+    val generatedJS = new JSTxBuilder()
+      .addQuery(NativeQuery("return db.FOO();"))
       .addQuery(NativeQuery("db.BAR();"))
       .generateJs()
 
-    generatedJS should be {
+    generatedJS shouldEqual {
```

Comment:
What is the reason for this change?

Reply:
There seems to be a functional difference between `equal` and `be`.
AS I understand it `be` always works the same as `==` in Scala, whereas `equal` will use a user-defined implicit `Equality` if available in scope. 

Maybe it would be better to use `be` as a default and `equal` only when we need the additional functionality?

https://www.scalatest.org/user_guide/using_matchers#checkingEqualityWithMatchers

Reply:
We could. I don't see too much practical difference to be honest, but I'm fine with either. I'm more care about consistency though, so that the same logic of choice is applied through the whole code base. We can discuss it on CQC.